meson.build

```diff
@@ -176,3 +176,8 @@ endif
 configure_file(input: 'tools/tuhi-gui-flatpak.py',
                output: 'tuhi-gui-flatpak.py',
                copy: true)
+
+tuhi_devenv = environment()
+tuhi_devenv.set('TUHI_DEVEL', meson.current_build_dir())
```

Comment:
This is a nitpick but: can we name this `TUHI_BUILD_DIR` instead? the _dir suffix for directory obviously, and builddir is better than devel which could also refer to the source directory.

Reply:
not quite sure I like `TUHI_BUILD_DIR` better. Just calling it that makes it seem like some kind of remnant of the build process rather than a way to run tuhi "portably". Would `TUHI_DEVEL_BUILD_DIR` work?

Reply:
bit clunky, but works. My main issue with `TUHI_DEVEL` is twofold:
- it doesn't specify that it's a *directory*, in plenty of other projects a `FOO_DEVEL` is a boolean to enable developer mode
- `DEVEL` to me points more at the sources than at the build output. But the latter is what we need.

Having said that, it's an internal thing, we're not going to really expose this so we can overthink the whole lot :) So `TUHI_DEVEL_BUILD_DIR` is fine.

Reply:
fixed